cmd/dump.go

```diff
@@ -17,6 +17,7 @@ import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/setting"
 	"github.com/Unknwon/cae/zip"
+	"github.com/Unknwon/com"
```

Comment:
Since `github.com/Unknwon/com` has been imported many place, I just used it. Or we can write a function about 10 lines.